src/System.Console/tests/SyncTextReader.cs

```diff
@@ -66,8 +66,15 @@ private static void Test(string content, Action action)
         }
         finally
         {
-            Console.SetOut(savedStandardOutput);
-            Console.SetIn(savedStandardInput);
+            using (TextWriter oldWriter = Console.Out)
```

Comment:
> I think you are doing this for an exception safe dispose on the oldWriter

Yes.  I can just remove the using if it'll make it clearer, and just call Dispose directly.